spec/_widget/components/app.spec.js

```diff
@@ -50,37 +50,25 @@ describe('App', () => {
   });
 
   describe('open', () => {
-    it('opens the support widget', async () => {
-      const subject = mount(App);
-      await subject.vm.open();
-
-      expect(subject.vm.isOpen).toEqual(true);
-      expect(subject.vm.$el.textContent).not.toContain(promptMessage);
-    });
-
-    it('pre-populates results based on the current URL', async () => {
```

Comment:
While I agree with the idea of this test, the test itself wasn't doing much helpful. I'll comb through this file more thoroughly when improving the search results.